.github/workflows/compressed.yml

```diff
@@ -0,0 +1,32 @@
+name: Compressed Size
+
+on:
+  push:
+    branches:
+      - master
+      - canary
+  pull_request:
+    branches:
+      - master
+      - canary
+  registry_package:
+    types: [published]
```

Comment:
Maybe this is preventing it from running? Because we aren't using the github package registry

Reply:
That's exactly what I was going to type to you :) I am checking this on my private repo once again and I'm sure that in a moment we will have proper commit here :)